nuxeo-arender-rendition-apb/apb.yml

```diff
@@ -0,0 +1,119 @@
+      display_group: Project properties
+
+    - name: arender_broker_image_pullPolicy
+      default: 'IfNotPresent'
+      type: enum
+      enum: ['IfNotPresent', 'Always', 'Never']
+      title: ARender Converter image pull policy
+      display_group: Container Specs
+
+    - name: arender_converter
+      type: string
+      title: ARender converter image
+      default: docker-arender.packages.nuxeo.com/arender-converter
+      display_group: Project properties
+
+    - name: arender_converter_image_pullPolicy
+      default: 'IfNotPresent'
+      type: enum
+      enum: ['IfNotPresent', 'Always', 'Never']
+      title: ARender Converter image pull policy
+      display_group: Container Specs
+
+    
+
+
+
+
+    - name: arender_rendition_image
+      type: string
+      title: ARender rendition image
+      default: docker-registry.default.svc:5000/common-infra/arender-rendition
+      display_group: Project properties
+
+    - name: arender_image_version
+      type: string
+      title: Image version
+      default: "1.0.1"
+      display_group: Project properties
+
+    - name: previewer_image_pullPolicy
+      default: 'IfNotPresent'
+      type: enum
+      enum: ['IfNotPresent', 'Always', 'Never']
+      title: ARender Previewer image pull policy
+      display_group: Container Specs
+    - name: previewer_podAnnotations
+      default: '{}'
+      type: string
+      title: Annotations to be added to ARender Previewer pods
+      display_type: textarea
+      display_group: Container Specs
+    - name: previewer_securityContext
+      default: '{}'
+      type: string
+      title: Previewer Security context for the pod
+      display_type: textarea
+      display_group: Container Specs
+    - name: previewer_resources
+      default: '{"limits": {"cpu": "2","memory": "2Gi"},"requests": {"cpu": "1","memory": "1Gi"}}'
+      type: string
+      title: Previewer Pod resource requests and limits
+      display_type: textarea
+      display_group: Container Specs
+
+    - name: rendition_image_pullPolicy
+      default: 'IfNotPresent'
+      type: enum
+      enum: ['IfNotPresent', 'Always', 'Never']
+      title: ARender Rendition image pull policy
+      display_group: Container Specs
+    - name: rendition_podAnnotations
+      default: '{}'
+      type: string
+      title: Annotations to be added to ARender Rendition pods
+      display_type: textarea
+      display_group: Container Specs
+    - name: rendition_securityContext
+      default: '{}'
+      type: string
+      title: Security context for the Rendition pod
+      display_type: textarea
+      display_group: Container Specs
+    - name: rendition_resources
+      default: '{"limits": {"cpu": "2","memory": "2Gi"},"requests": {"cpu": "1","memory": "1Gi"}}'
+      type: string
+      title: Rendition Pod resource requests and limits
+      display_type: textarea
+      display_group: Container Specs
```

Comment:
Same here, we have to use the splitted images definitions. I guess you want to do that later